apps/pie-storybook/stories/pie-modal.stories.ts

```diff
@@ -147,40 +151,42 @@ const createFocusableElementsPageHTML = () : TemplateResult => html`
 const BaseStoryTemplate = (props: ModalProps): TemplateResult => {
     const {
         aria,
+        dir,
         heading,
         headingLevel,
-        isDismissible,
         hasBackButton,
+        hasStackedActions,
+        isDismissible,
         isFooterPinned,
         isFullWidthBelowMid,
-        isOpen,
         isLoading,
+        isOpen,
+        leadingAction,
+        position,
         returnFocusAfterCloseSelector,
         size,
-        position,
         slot,
-        dir,
-        leadingAction,
         supportingAction,
     } = props;
     return html`
         <pie-button @click=${toggleModal}>Toggle Modal</pie-button>
         <pie-modal
+            .aria="${aria}"
+            dir="${dir}"
             heading="${heading}"
             headingLevel="${headingLevel}"
-            ?isDismissible="${isDismissible}"
             ?hasBackButton="${hasBackButton}"
+            ?hasStackedActions="${hasStackedActions}"
```

Comment:
I think for any property here that is not `reflected` we should use a `.` instead of a `?`. 

Using `?` is great for attributes as it means it won't be added if the condition is falsey. However for a `property` I think we'd prefer `.` which will set the value to the JS value passed in (so actual booleans or other js data types). 

I discovered this on my recent PR in which I tried using `?isFooterPinned=${isFooterPinned}` (our only default true boolean) which seemed to lead to some funky behaviour.

(I've shared a post about this internally if you want a bit more info!)

Reply:
Makes sense – I've made a ticket to investigate this so we can work out the best course of action going forward with these kind of props